2_Foundations/Visitors.md

```diff
@@ -97,7 +97,7 @@ This cascading simplifies implementations so they only need to override specific
 
 ## Traversal under your control
 
-By design none of the default apply(..) methods provided by vsg::Visitor and vsg::ConstVisitor undertake any type of traversal, the decision on which objects to traverse and how to traverse them is left to visitor subclasses.  The vsg::Object::traverse(..) method can be used by Visitor subclasses to handle traversal of an object's children when this is required, or Visitor subclasses can implement their own traversal of an object's children.
+By design none of the default apply(..) methods provided by vsg::Visitor and vsg::ConstVisitor undertake any type of traversal, the decision on which objects to traverse and how to traverse them is left to visitor subclasses. The vsg::Object::traverse(..) method can be used by Visitor subclasses to handle traversal of an object's children when this is required, or Visitor subclasses can implement their own traversal of an object's children.
```

Comment:
The space after a . is a readability aid, it helps give a sentence a bit more clarity,  As a dyslexic aids like this do help and why I put them in.

Reply:
Ah ok, the documentation is inconsistent in that regard (note that this was changed in lots of other places as well). Maybe some kind of 'documentation style guide' would help here?

Reply:
Well the changes to the VSG docs to remove spaces after . was also wrong but I'm so busy didn't have the energy to argue every single all of the time.